src/test/java/org/sagebionetworks/web/unitclient/widget/entity/tabs/ChallengeTabTest.java

```diff
@@ -50,20 +61,35 @@
 	CookieProvider mockCookieProvider;
 	ChallengeTab tab;
 	@Mock
-	AuthenticationController authenticationController;
+	AuthenticationController mockAuthenticationController;
 	@Mock
-	GlobalApplicationState globalApplicationState;
+	GlobalApplicationState mockGlobalApplicationState;
+	@Mock
+	EvaluationEditorReactComponentPage mockEvaluationEditorReactComponentPage;
```

Comment:
test that the evaluation editor react component page is configured and that the global editing state is flipped to true, and verify global editing state is flipped to false on callback.